src/test/java/uk/gov/justice/hmpps/prison/repository/OffenderBookingRepositoryTest.kt

```diff
@@ -138,19 +137,29 @@ class OffenderBookingRepositoryTest {
 
   @Test
   fun `test findWithSentenceSummaryByOffenderNomsIdAndBookingSequence returns correct number of sentences`() {
+    val bookings = repository.findAllByBookingIdIn(listOf(-3L))
+
+    assertThat(bookings.first { it.bookingId == -3L }.sentences).extracting<LocalDate> { it.sentenceStartDate }.containsOnlyOnce(
+      LocalDate.parse("2017-07-04"),
+      LocalDate.parse("2017-07-05"),
+    )
+  }
+
+  @Test
+  fun `check things`() {
     val agencyLocation = agencyLocationRepository.getReferenceById("LEI")
 
-    val bookings = repository.findAllOffenderBookingsByActiveTrueAndLocationAndSentences_statusAndSentences_CalculationType_CalculationTypeNotLikeAndSentences_CalculationType_CategoryNot(
+    val ids = repository.findDistinctByActiveTrueAndLocationAndSentences_statusAndSentences_CalculationType_CalculationTypeNotLikeAndSentences_CalculationType_CategoryNot(
       agencyLocation,
       "A",
       "BOB",
       "JOE",
-      Pageable.unpaged(),
+      PageRequest.of(0, 2, Sort.by("bookingId")),
     )
+    assertThat(ids.content).hasSize(2)
+    assertThat(ids.totalElements).isGreaterThanOrEqualTo(20)
```

Comment:
previously total elements was 24, with new distinct it is 22.  I've done greater than 20 though in case other tests add some more data